config/autoload/local.php.dist

```diff
@@ -1,3 +1,9 @@
 <?php
+
+use Zend\ConfigAggregator\ConfigAggregator;
+
 return [
+    // Cache the configuration: Recommended for production
+    // Closures used somewhere in the configuration make caching impossible
+    ConfigAggregator::ENABLE_CACHE => false,
```

Comment:
I see is _is_ in development.config.php.dist. You should probably decide where it should live and remove the other entry. Having it specified twice will confuse new users.